mets_mods2teiHeader/api/tei.py

```diff
@@ -111,6 +111,15 @@ def add_publisher(self, publisher):
         name.text = publisher
         publication_stmt.insert(0, publisher_node)
 
+    def add_source_edition(self, manuscript_edition):
+        """
+        Adds an edition statement with details on the source manuscript.
+        """
+        bibl_full = self.tree.xpath('//tei:fileDesc/tei:sourceDesc/tei:biblFull', namespaces=ns)[0]
```

Comment:
Will that xpath always return results? If not, an error with that xpath might be more intuitive than an IndexError.

Reply:
#22